crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -362,15 +362,55 @@ def nodes
 
       protected
 
+      # If there's separate network cluster, we have touch it before we start upgrade of other
+      # nodes, specificaly we need to evacuate the network routers from the first network node.
+      def prepare_network_node(network_node)
+        return if network_node.upgraded?
+
+        evacuate_network_node(network_node, network_node)
```

Comment:
Yeah, it reallly shouldn't matter much (apart from the .openrc). I guess the important part is that the neutronclient on the node that it is running on is on the same API level as the neutron-server (and even that might not be a real issue).